src/components/list/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {Crypto} from '../interfaces';
+import {
+  TextList,
+  Container,
+  CryptoLogo,
+  Line,
+  Arrow,
+  TextPercent,
+} from './styles';
+import ArrowUp from '../../images/icons/ArrowUp.png';
+import ArrowDown from '../../images/icons/ArrowDown.png';
+
+const List = ({cryptos}: {cryptos: Crypto[]}) => {
+  return (
+    <>
+      {cryptos.map((crypto: Crypto) => {
+        return (
+          <>
+            <Container container onlyPadding key={crypto.name}>
+              <Container container paddingLeftPercent>
+                <CryptoLogo source={crypto.img} />
+                <Container paddingLeft>
+                  <TextList bold={'bold'}> {crypto.name} </TextList>
```

Comment:
you should create a custom component for text, if you will use similar text design in the app, you are using a lot of props for the style